packages/itwinui-css/src/radio-tile/radio-tile.scss

```diff
@@ -134,9 +117,20 @@
 
   &:disabled:checked + * {
     z-index: 3;
-    border: 2px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+    outline: 2px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
     @media (forced-colors: active) {
-      border-color: GrayText;
+      outline-color: GrayText;
+    }
+  }
+
+  @include focus-visible {
+    + * { // outline for checkmark
+      z-index: 2;
+      outline-offset: -2px;
+      outline: 3px solid var(--iui-color-foreground-primary);
+      @media (forced-colors: active) {
+        outline-color: Highlight;
+      }
```

Comment:
`focus-visible` was moved so that its rules have precedence over the other rules.